scripts/fwup.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>fwup</id>
+    <title>fwup</title>
+    <version>0.9.1</version>
+    <authors>Frank Hunleth</authors>
+    <owners>Greg Mefford,Michael Schmidt</owners>
+    <summary>Configurable embedded Linux firmware update creator and runner</summary>
+    <description>The fwup utility is a configurable image-based firmware update utility for embedded Linux-based systems. It has two modes of operation. The first mode creates compressed archives containing root file system images, bootloaders, and other image material. These can be distributed via websites, email or update servers. The second mode applies the firmware images in a robust and repeatable way.</description>
+    <releaseNotes>https://github.com/fhunleth/fwup/releases</releaseNotes>
+    <projectUrl>https://github.com/fhunleth/fwup</projectUrl>
+    <tags></tags>
+    <packageSourceUrl>https://github.com/fhunleth/fwup</packageSourceUrl>
+    <docsUrl>https://github.com/fhunleth/fwup</docsUrl>
```

Comment:
I don't think it makes a big difference, but I think the `docsUrl` (as opposed to the `projectSourceUrl`) should be https://github.com/fhunleth/fwup/blob/master/README.md